### PR TITLE
chore: improving ci-cd process

### PR DESCRIPTION
**Pull request by justussoh**
This PR includes:
- renaming or steps in prod release process

### CODE REVIEW

File: .github/workflows/deploy_prod.yml [hunk 1]
```diff
@@ -39,15 +39,15 @@ jobs:
...
           expo-cache: true
       - uses: rlespinasse/github-slug-action@v2.x
-      - name: Parse Major Version
-        id: majorVersion
+      - name: Generate Release Channel
+        id: releaseChannel
         run: |
-          MAJOR_VERSION=$(echo ${{ env.GITHUB_REF_SLUG }} | sed -r 's/\.[0-9]+\.[0-9]+$//')
-          echo "::set-output name=majorVersion::$MAJOR_VERSION"
+          RELEASE_CHANNEL=$(echo ${{ env.GITHUB_REF_SLUG }} | sed -r 's/\.[0-9]+\.[0-9]+$//')
```
Comment (sebastianquek) [inline .github/workflows/deploy_prod.yml:45]: Just a minor thing, could you add a comment to showcase an example like the one you added on line 5?

Comment (justussoh) [reply .github/workflows/deploy_prod.yml:45]: added on line 42 as didnt want to put it in run!

File: .github/workflows/deploy_prod.yml [hunk 2]
```diff
@@ -60,7 +60,7 @@ jobs:
         uses: metcalfc/changelog-generator@v0.4.0
         with:
           myToken: ${{ secrets.GITHUB_TOKEN }}
-          base-ref: "prod-0" #need a tag "prod-0 for setup, please remove after setup"
+          # base-ref: "prod-0" #need a tag "prod-0 for setup, please remove after setup"
```
Comment (justussoh) [inline .github/workflows/deploy_prod.yml:63]: Commented out as already have a reference from the latest release

Comment (sebastianquek) [reply .github/workflows/deploy_prod.yml:63]: Let's delete this?

**Pull request merged**